index.md

```diff
@@ -0,0 +1,175 @@
+# GTFS-Realtime Data Best Practices
+
+## Introduction
+
+These are recommended practices for describing real-time public transportation information in the [GTFS Realtime](https://gtfs.org/reference/realtime/v2/) data format.

```

Comment:
One of six places where we use "real-time". Though that hyphenated presentation is the way it lives out in the universe a lot of the time, do we want to use it in the document when we're talking about "GTFS-realtime"?